### DISCUSSION TITLE
Error - need help with

### DISCUSSION THREAD

**Discussion by HunterAcosta** [chaiNNer-org/chaiNNer#2523] [General]
Hey all, just dloaded the latest version of ChaiNNer. I kept getting the same error in the last version and I have it in this new one. Never had this issue before hoping someone could help me troubleshoot or offer a bit of advice. Keep getting this error.

<img width="525" alt="Screenshot 2024-01-27 at 12 30 50" src="https://github.com/chaiNNer-org/chaiNNer/assets/44018596/bbddaf4c-478a-44cd-b49b-379d2040b5d6">

Also while im here I cant find the "Image File Iterator" has this been removed? its no longer in my tool bar.

Thanks 

Deano

edit* is it just that the file produced would be too large hence the buffer? again never had this issue before

Just cut the original image dimentions in half from 4000 px down to 200 and get the same error with smaller invalid buffer size

 
<img width="1393" alt="Screenshot 2024-01-27 at 12 51 23" src="https://github.com/chaiNNer-org/chaiNNer/assets/44018596/9a7202d0-5161-4cb0-9e91-c198872423ee">

**Comment by RunDevelopment** [comment]
> Also while im here I cant find the "Image File Iterator" has this been removed? its no longer in my tool bar.

It's called "Load Images" now.

> Keep getting this error.

Sounds like you don't have enough VRAM. How much VRAM does your GPU have?

Also, we recently updated to PyTorch 2.1. This version is generally faster, but seems to consume more VRAM.
